index.html

```diff
@@ -103,21 +103,23 @@
 
 ## Principles
 
-<dl>
-<dt><dfn id="oneweb" data-export="">
-  There is one web
-</dfn></dt>
-<dd>
+<section>
+  <h3 id="oneweb" data-export="">
```

Comment:
You should add `data-dfn-type=dfn` so this is still recognized as a definition, per https://tabatkins.github.io/bikeshed/#dfn-contract. Respec still doesn't handle internal [==] references to header definitions, but I think other specs will be able to find it.